definition-map.js

```diff
@@ -37,7 +51,7 @@ const QUERIES = {
             namespace: "The namespace is used to specify the content category for the translations. Accepts values of `lms` for platform content and `emails` for email content."
         }
     },
-    LanguagesQuery: {
+    Languages: {
```

Comment:
this might be intended for `Languages` instead?

Reply:
Yup, appreciate you catching that!